packages/react-examples/src/react-experiments/UnifiedPeoplePicker/DoubleUnifiedPeoplePicker.Example.tsx

```diff
@@ -179,14 +159,54 @@ const UnifiedPeoplePickerExample = (): JSX.Element => {
     setPeopleSelectedItems(updatedItems);
   };
 
-  const _onInputChange = (filterText: string): void => {
+  const SEPARATOR_REGEX = new RegExp(';|,|\\n|\u000A', 'g');
+  const splitPasteInputIntoRecipientStrings = (inputText: string) =>
+    inputText
+      .split(SEPARATOR_REGEX)
+      .map((token: string) => token.trim())
+      .filter((token: string) => !!token);
+
+  const _onInputChange = (filterText: string, composing?: boolean, resultItemsList?: IPersonaProps[]): void => {
     const allPeople = people;
+
+    const lastCharIndex = filterText.length - 1;
+    const lastChar = filterText[lastCharIndex];
     const suggestions = allPeople.filter((item: IPersonaProps) => _startsWith(item.text || '', filterText));
```

Comment:
I think these can be in the else condition? They're not used in the if right?

Reply:
it is used for the if check :) i will move it closer to its usage